src/Mapbender/WmsBundle/Element/WmsLoader.php

```diff
@@ -247,7 +247,7 @@ protected function getInstances()
                 $instances[] = $instConfig;
             }
         }
-        return new Response(json_encode(array("success" => json_encode($instances))), 200, array(
+        return new Response(json_encode(array("success" => $instances)), 200, array(
```

Comment:
Don't you mind to use here JsonResponse:
`return new JsonResponse(['success' => $instances]);` ? It takes a response status code as second argument, by default it's 200.

Reply:
Yeah, we can do this. I tried to do only minimal modification here, but I changed the line anyway so might as well do it right.